logdna/logdna.py

```diff
@@ -1,5 +1,6 @@
 import json
 import logging
+import platform
```

Comment:
I think you could probably remove this now that we're not writing platform details?